chromium_src/chrome/browser/ui/webui/extensions/extensions_ui.cc

```diff
@@ -25,6 +26,15 @@ void BraveAddExtensionsResources(content::WebUIDataSource* source,
                              IDS_EXTENSIONS_BRAVE_SPANNING_WARNING);
   source->AddLocalizedString("privateAndTorInfoWarning",
                              IDS_EXTENSIONS_BRAVE_PRIVATE_AND_TOR_WARNING);
+  source->AddLocalizedString("braveHosted", IDS_EXTENSIONS_BRAVE_HOSTED);
+
+  std::string mv2_extensions;
+  for (const auto& mv2_extension_id :
+       extensions_mv2::kPreconfiguredManifestV2Extensions) {
+    mv2_extensions += mv2_extension_id;
+    mv2_extensions += ",";
+  }
```

Comment:
Not 100% sure but can you use `base::JoinString(extensions_mv2::kPreconfiguredManifestV2Extensions, ",")` instead?

Reply:
good catch, I thought joinstring requires a std::vector and didn't re-check, but it takes a span, so yes it is much better to use JoinString